db/migrate/20210114064524_create_answers.rb

```diff
@@ -2,10 +2,14 @@ class CreateAnswers < ActiveRecord::Migration[5.2]
   def change
     create_table :answers do |t|
       t.integer :lesson_id, foreign_key: true
-      t.integer :choices_id, foreign_key: true
+      t.integer :choice_id, foreign_key: true
       t.integer :word_id, foreign_key: true
 
       t.timestamps
     end
+
```

Comment:
Delete the space here. Keep your migration file clean or else it might affect your Database.